tests/wicked/basic/sut/t03_static_addresses_xml.pm

```diff
@@ -26,7 +26,7 @@ sub run {
     $self->get_from_data('wicked/static_address/static-addresses.xml', $config);
     file_content_replace($config, '--sed-modifier' => 'g', xxx => $ctx->iface());
     $self->wicked_command('ifup --ifconfig /data/static_address/static-addresses.xml', $ctx->iface());
-    $self->assert_wicked_state(ping_ip => '10.0.2.2', iface => $ctx->iface());
+    $self->ping_with_timeout(type => 'host', interface => $ctx->iface());
```

Comment:
please check `assert_wicked_state` function it is doing more than just ping , are you sure you want to drop all this ? I would recommend replace one line inside assert_wckied_state function

Reply:
I will keep assert_wicked_state in this PR and just change the ping_ip.